.travis.yml

```diff
@@ -1,21 +1,43 @@
 language: dart
 
 dart:
-- 2.3.0
 - dev
 
-dart_task:
-- test: -p vm
-- dartanalyzer: --fatal-infos --fatal-warnings .
+jobs:
+  include:
+    - stage: analyze_and_format
+      name: "Analyze lib/ (no experiment flag)"
+      dart: dev
+      os: linux
+      script: dartanalyzer --fatal-warnings --fatal-infos lib/
```

Comment:
I should have noticed this earlier, this step is redundant now with the one below.

Reply:
#161